### PR TITLE
bash: add Polish translation

### PR DESCRIPTION
**Pull request by maslo-sh**
bash: add Polish page

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): 3.2.57(1)-release**

### CODE REVIEW

File: pages.pl/linux/bash.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- `-e` Wykonaj przekazany skrypt do pojawienia się pierwszego błędu:
+
+  `bash -e {{sciezka/do/skrypt.sh}}`
+
+- Wykonaj komendę przekazaną przez strumień wejścia `stdin`:
+
+  `{{echo "echo 'bash uruchomiony'"}} | bash`
+
+- `-r` Uruchom sesję Bash w trybie [r]estrykcyjnym (więcej o trybie restrykcyjnym na stronie <https://www.gnu.org/software/bash/manual/html_node/The-Restricted-Shell.html>).
```
Comment (Velox0) [inline pages.pl/linux/bash.md:34]:
```suggestion
- `-r` Uruchom sesję Bash w trybie [r]estrykcyjnym (więcej o trybie restrykcyjnym na stronie <https://www.gnu.org/software/bash/manual/html_node/The-Restricted-Shell.html>):
```

**Review by Velox0 — CHANGES_REQUESTED** [review]

File: pages.pl/linux/bash.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  `bash -x {{sciezka/do/skrypt.sh}}`
+
+- `-e` Wykonaj przekazany skrypt do pojawienia się pierwszego błędu:
+
+  `bash -e {{sciezka/do/skrypt.sh}}`
+
+- Wykonaj komendę przekazaną przez strumień wejścia `stdin`:
+
+  `{{echo "echo 'bash uruchomiony'"}} | bash`
```
Comment (kbdharun) [inline pages.pl/linux/bash.md:32]:
```suggestion
`bash --norc`

- `-c` Wywołaj określoną komendę w powłoce Bash:

`bash -c "{{komenda powłoki systemowej}}"`

- Uruchom przekazany jako argument skrypt shella:

`bash {{sciezka/do/skrypt.sh}}`

- `-x` Wykonaj przekazany jako argument skrypt, wypisując wszystkie wykonane w procesie komendy na standardowe wyjście `stdout`:

`bash -x {{sciezka/do/skrypt.sh}}`

- `-e` Wykonaj przekazany skrypt do pojawienia się pierwszego błędu:

`bash -e {{sciezka/do/skrypt.sh}}`

- Wykonaj komendę przekazaną przez strumień wejścia `stdin`:

`{{echo "echo 'bash uruchomiony'"}} | bash`
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution, can you please sign the CLA above for this PR to go through?

File: pages.pl/linux/bash.md
[the remark this answers is not in the document]
Comment (maslo-sh) [reply pages.pl/linux/bash.md:1]: Done.

**Comment by kbdharun** [comment]
@acuteenvy Feel free to take over/close this PR, since the author has been unresponsive for a while.

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**